src/components/ViewArticle/Comments.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../../utils/api";
+
+const Comments = ({ article }) => {
+  const [comments, setCommments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getCommentsByArticleId(article.article_id).then((comments) => {
+      setCommments(comments);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) return <p>Loading Comments...</p>;
+  return (
+    <section className="comments-section">
+      {comments.map((comment) => {
+        return (
+          <div>
+            <h1>{comment.author}</h1> {comment.body}
```

Comment:
Don't forget the rule in HTML: Only one H1 per page. Maybe you could change this to an H2 instead?